### ISSUE TITLE
FMX: click to center motions are 2x

### ISSUE THREAD

**Issue by JunAishima** [NSLS-II/lsdc#120]
on FMX, click to center causes motions of the sample to be doubled - click on the sample, it moves twice as far, so ends up at the position exactly the same distance away from the crosshairs.

Also fails with "Define Center".

Seems to work correctly (1x movement) when trying to center on a sample below and to the right of the crosshairs?

Reported by Wuxian. ROIs do not appear to be the issue. sample x motions done directly through CSS look like it matches the distance shown in the scale (when the motor is moved that much)

**Comment by JunAishima** [comment]
noticed that the logs were repeating all of their messages - this pointed to there being two servers running. "ps -ef lsdcServer" found two servers. After the scan finished, I looked at /usr/local/bin/lsdcServer and noticed that the command to kill other daqs was not there.

Updated kill_daq so that it will use pkill, which can find processes based on any part of the call - killall seems to only care about the direct command used.
